pkg/manifest/mantaray/marshal.go

```diff
@@ -83,6 +83,8 @@ var (
 	ErrInvalidInput = errors.New("input invalid")
 	// ErrInvalidVersionHash signals unknown version of hash.
 	ErrInvalidVersionHash = errors.New("invalid version hash")
+	// ErrInvalidOutput signals when malformed manifest contnet is supplied to Unmarshal function
+	ErrInvalidOutput = errors.New("malformed manifest contents")
```

Comment:
I would rather call it `ErrInvalidManifest`. Output doesnt make much sense.

Reply:
Agree, also the error string should match the error name.

Reply:
@notanatol, do you agree to use @mrekucci and @aloknerurkar suggestion for this error variable?

Reply:
sure, sounds good